CHANGELOG.md

```diff
@@ -11,9 +11,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - handle GET EntitySet payload without the member results - Jakub Filak
+- both Literal and JSON DateTimes has Timezone set to UTC - Jakub Filak
 
 ### Fixed
 - removed superfluous debug print when parsing FunctionImports from metadata - Jakub Filak
+- use correct type of deserialization of Literal (URL) structure values - Jakub Filak
+- null values are correctly handled and nullable configuration is honored - Jakub Filak
```

Comment:
small note: does that means that actually line 19 is the topic of this PR and line 14 is another feature in same PR? 
only provoking a bit :)

Reply:
> small note: does that means that actually line 19 is the topic of this PR and line 14 is another feature in same PR?
only provoking a bit :)

It makes no sense to open a new PR for every small change especially when the little changes is blocking a bigger change. The unification of timezones is not a feature at all - it's rather a bug fix.

> But for real: Should't the conversion be configurable with UTC as sane default?

I am going to add the constant you mentioned somewhere. If you want to make it configurable, I am happy to review your Pull Request with that change.

Reply:
Done, have a look at the first commit. I added the function current_timezone().